sparkbox/sparkbot
# Catchup with the times

This is a big PR. It takes a lot of baby steps to get all dependencies up to the latest, including hubot itself. We can clean up the commits at the end.

At the end of the day, this PR removes unused (or at least very unpopular) hubot scripts, updates dependencies, and simplifies the startup of Hubot. 

### Testing Locally
1. `npm install`
2. `npm start`

From here, make sure you can interact with sparkbot via things like `sparkbot help` and `sparkbot lnl list`.

### Testing against Slack
Review apps are available for Sparkbot, but you have to realize 2 instances listening. This can be confusing for real users, so make sure to delete the review app soon after testing.

1. Head to the [sparkbox-hubot Heroku Pipeline](https://dashboard.heroku.com/pipelines/80535dd2-1889-41fa-8cff-c0f8788cc6ff).
2. Click _Create Review App_ for PR #17. 
3. Wait for it to build.
4. Head to Sparkbox Slack and start a DM with `@sparkbot`
5. Make sure you can interact with `@sparkbot`. 

⚠️ Be careful using data modification commands like `lnl forget 2` since both the PR and real version of sparkbot have different data in spot 2.
